Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -57,6 +57,7 @@ final class NoteOptionsViewController: UITableViewController {
     func setupViewStyles() {
         tableView.backgroundColor = .simplenoteTableViewBackgroundColor
         tableView.separatorColor = .simplenoteDividerColor
+        publishActivityIndicator.color = UIColor.color(name: .tableViewDetailTextLabelColor)
```

Comment:
@jleandroperez Tinting the activity indicator is better as it respects the theme changes. Is it worth adding a dedicated colour for it to UIColor+Studio or is using another colour like this okay? I went for the detail text label as the indicator is in the same place that label would be so it keeps the same styling.

Reply:
Oh not really!! I was actually thinking about iOS 13 dynamic colors. The `tableViewDetailTextLabelColor` case isn't iOS 13 friendly. **However, just tested, and it's reacting well to trait changes**, so no actual need to change anything!!

Apologies about the noise!!